src/firebase/firebase.test.ts

```diff
@@ -50,6 +50,7 @@ describe(initializeCloudMessaging, () => {
     let catchedError
 
     await expectSaga(initializeCloudMessaging, app, mockAccount)
+      .dispatch({ type: 'HOME/VISIT_HOME' })
```

Comment:
these tests fail with a timeout without the home screen visit action. I wasn't able to turn that into an automated test for reasons discussed previously, but between this and the manual test result, I'm comfortable moving forward anyway.